guides/source/getting_started.md

```diff
@@ -422,7 +422,7 @@ create something with a bit more substance.
 In the Blog application, you will now create a new _resource_. A resource is the
 term used for a collection of similar objects, such as articles, people, or
 animals.
-You can create, read, update, and destroy items for a resource and these
+You can create, read, update, and destroy items in a resource and these
```

Comment:
Don't think this is right. I wouldn't say articles are in an author, for example. Comments can be expressed as in a post, but for seems more appropriate and more generally suitable.

Reply:
Ah I see what you mean! Thank you for explaining 🙂